src/gui/MatchTemplatePanel.cpp

```diff
@@ -703,7 +707,11 @@ void MatchTemplatePanel::StartEstimationClick(wxCommandEvent& event) {
         number_of_rotations++;
     }
 
-    current_job_package.Reset(active_refinement_run_profile, "match_template", number_of_jobs);
+    // CPU match_template can probably be DEPRECATED
+    if ( use_gpu )
+        current_job_package.Reset(active_refinement_run_profile, "match_template_gpu", number_of_jobs);
```

Comment:
Ah, I see. Thats how the gui switches to the new exectuable

Reply:
yep, good old const char!